scripts/generate-babel-types-docs.js

```diff
@@ -4,7 +4,7 @@ var util = require('util');
 var path = require('path');
 var fs = require('fs');
 
-var types = require('../packages/babel-types');
+var types = require('../packages/babel-types/src');
```

Comment:
It's probably because it's a separate script so we assume node 6 while for other code we just run Babel on it